SCClassLibrary/Common/Control/Server.sc

```diff
@@ -670,36 +578,53 @@ Server {
 		^Bus(\audio, 0, this.options.numOutputBusChannels, this);
 	}
 
-	startAliveThread { arg delay=0.0;
-		this.addStatusWatcher;
-		^aliveThread ?? {
-			aliveThread = Routine({
-				// this thread polls the server to see if it is alive
-				delay.wait;
-				loop({
-					this.status;
-					aliveThreadPeriod.wait;
-					this.serverRunning = alive;
-					alive = false;
-				});
-			});
-			AppClock.play(aliveThread);
-			aliveThread
-		}
+
+	/* backward compatibility */
```

Comment:
Ah yes. I was wondering whether it is OK to have the getter to the watcher. I used it once:

````
*resumeThreads {
		set.do { |server| server.statusWatcher.resumeThread }
	}
````

I could of course add a method to Server that isolates it. What do you think?